modin/pandas/test/dataframe/test_udf.py

```diff
@@ -160,11 +160,6 @@ def apply_func(series, y):
     )
 
 
-@pytest.mark.xfail(
-    reason="Modin's 'apply' produces DataFrame instead of Series and so insertion"
-    "of the apply result to the source frame is failed. Issue to track this bug:"
-    "https://github.com/modin-project/modin/issues/3219"
-)
```

Comment:
if that is fixed why the issue is still open?

Reply:
That is not fixed. We still have a problem described in #3219. This test checks kind of another thing.

Reply:
Is enabling the test related to pandas migration? 😛

Reply:
Yep :)

Reply:
I didn't look into it.